src/odemis/driver/test/technolution_test.py

```diff
@@ -132,7 +132,7 @@ def test_convertBits(self):
             # Due to uneven scaling an absolute tolerance of 0.5 is allowed.
             # (uneven scaling means that 0.0 is not mapped to zero but to 0.5 due to the uneven range of INT16)
             tolerance = 0.5
-            expeceted_value = test_value * INT16.max
+            expeceted_value = test_value * 2**15 - 1
             if not almost_equal(expeceted_value[0], out[0], atol=tolerance):
```

Comment:
Now that I'm paying more attention to this specific test case, I'm not convinced that it's testing the right thing. It seems it's testing 19*2 times the same thing. I have a feeling you could test *more* by just something like:
```python
out = convert2Bits((-10, 0, 10), (-10, 10))
self.assertEqual(tuple(out), (-2**15, -1, 2**15-1))

out = convert2Bits((0, 0.5, 1), (0, 1))
self.assertEqual(tuple(out), (-2**15, -1, 2**15-1))
```
This assumes the function returns values already floored to an int.